backend/apps/chats/serializers.py

```
@@ -0,0 +1,258 @@
+
+    def get_members(self, obj):
+        return obj.memberships.count()
+        
+    def get_created_by_email(self, obj):
+        return obj.created_by.email if obj.created_by else None
+
+
+class ChatGroupCreateUpdateSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = ChatGroup
+        fields = ["name", "description", "label", "can_members_leave"]
```

Ruff (0.15.5)

[warning] Mutable default value for class attribute

(RUF012)